Twig/Extension/BlockExtension.php

```diff
@@ -36,6 +36,9 @@ public function __construct(BlockHelper $blockHelper)
     public function getFunctions()
     {
         return array(
+            new \Twig_SimpleFunction('sonata_block_exists',
```

Comment:
Please follow PSR-2 standard: https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-2-coding-style-guide.md#46-method-and-function-calls